src/GraphNodeManagerViewExtension/GraphNodeManagerViewModel.cs

```diff
@@ -1,24 +1,23 @@
-using System;
-using System.Collections.Generic;
-using System.Collections.ObjectModel;
-using System.Linq;
-using System.Windows.Data;
-//using System.Windows.Input;
 using Dynamo.Core;
+using Dynamo.Extensions;
 using Dynamo.Graph.Nodes;
 using Dynamo.Graph.Workspaces;
+using Dynamo.GraphNodeManager.Properties;
 using Dynamo.GraphNodeManager.ViewModels;
 using Dynamo.Models;
-using Dynamo.Wpf.Extensions;
-using Dynamo.Extensions;
-using Dynamo.GraphNodeManager.Properties;
 using Dynamo.PackageManager;
 using Dynamo.Utilities;
 using Dynamo.ViewModels;
+using Dynamo.Wpf.Extensions;
+using Newtonsoft.Json;
 using Prism.Commands;
+using System;
```

Comment:
Please adjust follow Dynamo coding standards instead of touching the order in your PR, please refer to https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards#usings